### PR TITLE
[AEA-1007] Expand known issues section of quickstart

### PR DESCRIPTION
**Pull request by marcofavorito**
## Proposed changes

Expand known issues section of quickstart, including links to installation instructions.

## Fixes

n/a

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have checked that the documentation about the `aea cli` tool works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/quickstart.md [hunk 1]
```diff
@@ -62,6 +62,22 @@ sudo apt-get install python3.7-dev
 
 - Windows users: install <a href="https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019" target="_blank">tools for Visual Studio</a>.
 
+- To install Go, follow the
+ official guide, depending on your platform: 
+ [https://golang.org/doc/install](https://golang.org/doc/install)
+
+- Python is already included by default on
```
Comment (DavidMinarsch) [inline docs/quickstart.md:69]: Higher up there is already something on Ubuntu/Debian. Is that 3.7 specific?

Comment (marcofavorito) [reply docs/quickstart.md:69]: the thing above is about Python headers, that are not included by default. Here we are only talking about the interpreter.

File: docs/quickstart.md [hunk 2]
```diff
@@ -62,6 +62,22 @@ sudo apt-get install python3.7-dev
```
Comment (marcofavorito) [inline docs/quickstart.md:62]: actually, on Ubuntu LTS 20.04 the default version is 3.8, and on Ubuntu LTS 18.04 the default is 3.7. Maybe we could list the two cases.

File: docs/quickstart.md (hunk 1 above)
Comment (DavidMinarsch) [reply docs/quickstart.md:69]: True

File: docs/quickstart.md (hunk 2 above)
Comment (DavidMinarsch) [reply docs/quickstart.md:62]: I mean why not just say do it for the version which is relevant to you (i.e. first check version, then run appropriate command)

**Review by DavidMinarsch — APPROVED** [review]
LGTM

**Pull request merged**